.github/workflows/automation-benchmark-tests.yml

```diff
@@ -60,6 +64,7 @@ jobs:
       TEST_INPUTS: ${{ inputs.TestInputs }}
       TEST_TRIGGERED_BY: automation-benchmark
       CHAINLINK_ENV_USER: ${{ github.actor }}
+      ENV_JOB_IMAGE: ${{ secrets.QA_AWS_ACCOUNT_NUMBER }}.dkr.ecr.${{ secrets.QA_AWS_REGION }}.amazonaws.com/chainlink-tests:${{ github.head_ref || github.ref_name }}
```

Comment:
```suggestion
      ENV_JOB_IMAGE: ${{ secrets.QA_AWS_ACCOUNT_NUMBER }}.dkr.ecr.${{ secrets.QA_AWS_REGION }}.amazonaws.com/chainlink-tests:${{ env.BRANCH_NAME }}
```

Reply:
This suggestion is invalid due to env not being present when setting env values: https://github.com/smartcontractkit/chainlink/actions/runs/4089213445